contracts/interfaces/IAelinUpFrontDeal.sol

```diff
@@ -74,7 +74,14 @@ interface IAelinUpFrontDeal {
         uint256 totalDealTokens
     );
 
-    event ClaimDealTokens(address indexed user, uint256 amountMinted, uint256 amountPurchasingReturned);
+    event ClaimVestingToken(
```

Comment:
we should be wary of changing the names of events due to the subgraph issues. We can change the naming convention in the UI without touching this